rsmtool/container.py

```diff
@@ -1,6 +1,11 @@
 """
-Classes for storing any kind of data contained
-in a pd.DataFrame object.
+Class to encapsulate data contained in multiple pandas DataFrames.
+
+It represents each of the multiple data sources as a "dataset". Each
+dataset is represented by three things:
```

Comment:
Maybe a different word than "thing" here would be better.
```suggestion
dataset is represented by three properties:
```

I see that "keys" is used below. "fields" is another option.

Reply:
Much better!